src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs

```diff
@@ -63,21 +65,21 @@ int Foo
         [WorkItem(6034, "https://github.com/dotnet/roslyn/issues/6034")]
         public async Task TestMethodWithArrowBody()
         {
-            await TestAsync(
+            await TestWithAllCodeStyleOff(
 @"class C
 {
     int [||]GetFoo() => 0;
 }",
 @"class C
 {
-    int Foo => 0;
+    int Foo { get { return 0; } }
```

Comment:
a test was added to verify that if the user wants expression bodied properties that this will work properly.